ui/src/components/Publication/Card/index.tsx

```diff
@@ -23,7 +24,22 @@ const Card: React.FC<Props> = (props): React.ReactElement => (
                     : props.publication.title}
             </p>
             <span className="mb-4 block font-montserrat text-sm text-grey-800 transition-colors duration-500 dark:text-white-50">
-                {props.publication.user.firstName[0]}. {props.publication.user.lastName}
+                {props.publication.user.id === 'octopus' ? (
+                    <>
+                        {props.publication.user.firstName[0]}. {props.publication.user.lastName}
```

Comment:
at some point would be nice to wrap this in a component, we have the same code repeated 7 or 8 times